test/e2e/common/runtime.go

```diff
@@ -277,7 +277,7 @@ while true; do sleep 1; done
 				},
 				{
 					description: "should be able to pull image from docker hub",
-					image:       "alpine:3.7",
+					image:       "golang:latest",
```

Comment:
Agree with @neolit123, can we not use a :latest at least?
Ideally we want to pull the smallest viable image as well.